rest/utilities_testing.go

```diff
@@ -1320,17 +1326,77 @@ func createBlipTesterWithSpec(tb testing.TB, spec BlipTesterSpec, rt *RestTester
 		return nil, err
 	}
 
+	collections := bt.restTester.getCollectionsForBLIP()
+	if !spec.skipCollectionsInitialization && len(collections) > 0 {
+		bt.initializeCollections(collections)
+	}
```

Comment:
The interaction between skipCollectionsInitialization and getCollectionsForBLIP doesn't seem like it covers all the cases.  It seems like we'd need the ability to test:

1. named collections without getCollections message (negative test)
2. named collections with getCollections message (positive test)
3. default collection only without getCollections message (positive test)
4. default collection only with getCollectionsMessage (positive test)
It looks to me like number 4 isn't supported with this approach?

Reply:
In general we only test 1 and 4 in a specific test wit h`skipInitialization` in `TestBlipGetCollections` where we `skipCollectionInitialization`.